src/main/java/uk/gov/hmcts/reform/hmc/controllers/HearingActualsManagementController.java

```diff
@@ -28,14 +29,16 @@ public HearingActualsManagementController(HearingActualsService hearingActualsSe
         produces = APPLICATION_JSON_VALUE)
     @ResponseStatus(HttpStatus.OK)
     @ApiResponses(value = {
-        @ApiResponse(code = 200, message = "Hearing actuals update processed"),
-        @ApiResponse(code = 400, message = "Invalid hearing Id"),
-        @ApiResponse(code = 400, message = "001 No such id: hearingId"),
-        @ApiResponse(code = 400, message = "002 invalid status HEARING_REQUESTED"),
-        @ApiResponse(code = 400, message = "002 invalid status AWAITING_LISTING"),
-        @ApiResponse(code = 400, message = "003 invalid date"),
-        @ApiResponse(code = 400, message = "004 non-unique dates"),
-        @ApiResponse(code = 500, message = "Error occurred on the server")
+        @ApiResponse(code = 200, message = ValidationError.HEARING_ACTUALS_SUCCESSFULLY_PROCESSED),
```

Comment:
for all successful flows can they be reverted back to a string and removed from Validation.Error